content/id/docs/concepts/cluster-administration/monitoring.md

```diff
@@ -0,0 +1,167 @@
+juga mengekspos metrik-metrik dalam _endpoint-endpoint_ seperti `/metrics/cadvisor`, 
+`/metrics/resource` dan `/metrics/probes`. Metrik-metrik tersebut tidak memiliki
+siklus hidup yang sama.
+
+Jika klaster kamu menggunakan {{< glossary_tooltip term_id="rbac" text="RBAC" >}}, 
+membaca metrik-metrik itu memerlukan otorisasi melalui sebuah User, Group, atau 
+ServiceAccount dengan ClusterRole yang memungkinkan untuk mengakses `/metrics`.
+
+Sebagai contoh:
+
+```
+apiVersion: rbac.authorization.k8s.io/v1	
+kind: ClusterRole	
+metadata:	
+  name: prometheus	
+rules:	
+  - nonResourceURLs:	
+      - "/metrics"	
+    verbs:	
+      - get
+```
+
+## Siklus Hidup (_Lifecycle_) Metrik
+
+Alpha metric →  Stable metric →  Deprecated metric →  Hidden metric → Deletion
+
+Metrik-metrik _Alpha_ tidak memiliki jaminan stabilitas; dengan demikian mereka 
+dapat dimodifikasi atau dihapus kapan saja.
+
+Metrik-metrik _Stable_ dijamin tidak berubah; Secara khusus, stabilitas berarti:
+
+* metrik itu sendiri tidak akan dihapus (atau diganti namanya)
+* jenis metrik tidak akan dimodifikasi
+
+Metrik _Deprecated_ memberi sinyal bahwa metrik akan segera dihapus; untuk 
+menemukan versi mana, kamu perlu memeriksa anotasi, yang mencakup dari versi 
+Kubernetes mana yang metrik tersebut akan dianggap _deprecated_.
+
+Sebelum _deprecation_:
+
+```
+# HELP some_counter this counts things
+# TYPE some_counter counter
+some_counter 0
+```
+
+Sesudah _deprecation_:
+
+```
+# HELP some_counter (Deprecated since 1.15.0) this counts things
+# TYPE some_counter counter
+some_counter 0
+```
+
+Setelah metrik disembunyikan maka secara default metrik tidak dipublikasikan 
+untuk _scraping_. Untuk menggunakan metrik yang tersembunyi, kamu perlu mengganti 
+konfigurasi untuk komponen klaster yang relevan.
+
+Setelah metrik dihapus, metrik tidak dipublikasikan. Kamu tidak dapat mengubah 
+ini menggunakan _override_.
+
+## Melihat Metrik-metrik tersembunyi
+
+Seperti dijelaskan di atas, admin-admin dapat mengaktifkan metrik tersembunyi 
+melalui tanda pada baris perintah untuk binari tertentu. Ini bermaksud untuk 
+digunakan sebagai jalan keluar bagi admin-admin apabila mereka gagal memigrasi 
+metrik yang sudah _deprecated_ dalam rilis terakhir.
+
+Tanda `show-hidden-metrics-for-version` menunjukkan versi yang ingin kamu tampilkan 
+metrik _deprecated_-nya dalam rilis tersebut. Versi ini ditunjukkan sebagai x.y, 
+di mana x adalah versi major, y ​​adalah versi minor. Versi _patch_ tidak 
+diperlukan meskipun metrik dapat di_deprecate_ dalam rilis _patch_, alasan untuk
+itu adalah kebijakan _deprecation_ metrik berlaku pada rilis minor.
+
+Tanda hanya dapat menggunakan versi minor sebelumnya sebagai parameternya. Semua 
+metrik yang disembunyikan di sebelumnya akan dikeluarkan jika admin-admin 
+mengatur versi sebelumnya ke `show-hidden-metrics-for-version`. Versi yang 
+terlalu lama tidak diizinkan karena hal ini melanggar kebijakan metrik yang
```

Comment:
```suggestion
terlalu lama tidak diizinkan karena hal ini melanggar kebijakan _deprecation_ metrik.
```

The original text says "... because this violates the metrics deprecated policy."
The "metrics deprecated policy" here is confusing, as the previous paragraph talks about "metrics deprecation policy". I think we can change it for clarity here.

The "deprecated policy" also occurs in the next paragraph.
Is this a bug? CMIIW @sftim 
@ariscahyadi

Reply:
For me those are two different things:
`metric deprecated policy` means policy for the deprecated metric
`metric deprecation policy` means policy to deprecate the metric

CMIIW :)

Reply:
I think this sentence relates to the metrics lifecycle
![image](https://user-images.githubusercontent.com/3192347/79633114-98abf700-818d-11ea-9157-a85613f3b6b7.png)

if so, this particular sentence means "policy when to deprecate the metric",  cmiiw

Reply:
Any solutions for this? I prefer this:

`metrics deprecation policy` -> `kebijakan _depecration_ metrik`
`metrics deprecated policy` -> `kebijakan untuk metrik yang sudah _deprecated_`

Reply:
I used the above preference.